packages/nvidia_nat_mcp/scripts/check_mcp_auth_jwt.py

```
@@ -0,0 +1,340 @@
+import sys
+import time
+import webbrowser
+from urllib.parse import urljoin
+
+import httpx
+import websockets
+
+try:
+    from authlib.jose import jwt
+except ImportError as e:
+    raise ImportError("authlib is required for check_mcp_auth_jwt. Install with: pip install authlib") from e
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)